WalletWasabi/Exceptions/NoAvailableCoinsException.cs

```diff
@@ -0,0 +1,8 @@
+namespace WalletWasabi.Exceptions;
+
+public class NoAvailableCoinsException : Exception
```

Comment:
It only makes sense to have a special exception if you catch that special exception somewhere.

Reply:
Its catch-block got deleted. I will remove this.